e2e/utils/waits-utils.ts

```diff
@@ -401,7 +408,6 @@ export async function waitWhileLoading(
       ),
       page.waitForSelector(spinElementsSelector, { hidden: true, timeout })
     ]);
-    await page.waitForTimeout(500);
```

Comment:
Hoping tests do not need a half second sleep. It will make tests run faster. I will monitor for flaky failures.